### ISSUE TITLE
Repeated fields are not implementing toObject()

### ISSUE THREAD

**Issue by zeidoo** [thesayyn/protoc-gen-ts#7]
Hi,

I've been trying to use this with repeated fields,
```
message Schema {
    string name = 1;
    repeated Field fields = 2;
}
```

The generated code fails to compile:
```        
get fields(): Field[] {
            return pb_1.Message.getRepeatedWrapperField(this, Field, 2) as Field[];
        }
        set fields(value: Field[]) {
            pb_1.Message.setRepeatedWrapperField(this, 2, value);
        }
        toObject() {
            return {
                name: this.name,
                fields: this.fields && this.fields.toObject() <-- the fields array doesn't implement toObject, the individual items themselves do.
            };
        }
```

The error when compiling is: Property 'toObject' does not exist on type 'Field[]'.

**Comment by thesayyn** [comment]
I can confirm the issue. As far as I understand it should generate this `field.map(m => m.toObject())` instead of `field.toObject()`.

 I would accept a PR with appropriate tests if you are willing to help. 😊

**Comment by zeidoo** [comment]
Yup, I'll PR it sometime in the coming week.

**Comment by thesayyn** [comment]
Hey @zeidoo. Have you got any free time to work on this?

**Comment by thesayyn** [comment]
Released in 0.0.6. @zeidoo FYI
